dir.targets

```diff
@@ -56,4 +56,11 @@
     </SupplementalTestData>
   </ItemGroup>
 
+  <Target Name="ProducesPackageId" 
```

Comment:
Yes, as previously mentioned, this will move to buildtools after it's fleshed out and we are ready to bring other repo's onboard which have similar build infrastructure (like WCF).